### DISCUSSION TITLE
Velocity View Issues / To Do

### DISCUSSION THREAD

**Discussion by seangoodvibes** [SynthstromAudible/DelugeFirmware#2566] [Ideas]
### Please describe the problem:

Using this issue to document issues as I discover them with velocity view:

- [x] automation overview for kit clip with affect entire off seems to react to shortcut presses even when the message displayed on the screen says "Select Affect Entire or Kit Row" - this can lead to shortcuts blinking incorrectly
- [x] kit clip with affect entire off does not show automation overview with only the velocity shortcut lit up if the drum type selected is MIDI or CV.
- [x] velocity lanes don't get rendered for kit rows of drum type MIDI or CV if affect entire is off (lanes do get rendered if affect entire is on)
- [x] when entering a long press, the last released pad sets the default velocity. it should be the second pressed pad that sets the default velocity.
- [x] LastAuditionedYDisplay can become out of sync with selected drum in a kit
- [ ] Use Affect Entire toggle to switch between editing Velocity of all note rows vs Velocity of one note row. When editing velocity of all note rows, only adjust existing velocity (remove functionality to add / remove notes)
- [ ] use the clip doubling shortcut (Shift + press <>) to double just the note row selected
- [ ] add pad selection mode

### What is the expected behavior?

Above issues are fixed

### What hardware did you reproduce it with?

OLED

### What firmware did you reproduce it with?

Built from community

**Comment by m-m-adams** [comment]
Gonna move this to discussion since the remaining ones aren't really problems
